### PR TITLE
Add exts into DAP script to capture json downloads

### PR DESCRIPTION
**Pull request by tdlowden**
As per DAP [Implementation Guide](https://www.digitalgov.gov/files/2015/02/GSA-DAP-UA-Code-Quick-Guide-15-01-30-v1-02_mvf.pdf), this should allow for recording of the json downloads.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -322,5 +322,5 @@ <h2 id="current_visitors" class="data">...</h2>
   <script src="/js/index.js"></script>
 
   <!-- report to oneself -->
-  <script id="_fed_an_ua_tag" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=GSA"></script>
+  <script id="_fed_an_ua_tag" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=GSA&exts=json"></script>
```
Comment (konklone) [inline index.html]: This is a syntax error, there should be an `&` before `ext=json`, not a space.

Can you fix that, test the event locally to make sure it's firing as expected, and then ping the PR to say it's working?

Comment (tdlowden) [inline index.html]: Whoa my fault. I mis-typed that. Will fix.

**Comment by tdlowden** [comment]
Weird, my original syntax was in fact correct, with an &. I can see it in the branch file. But the diff removes it. Where have I gone wrong here?

**Comment by konklone** [comment]
Ah, it **is** there, it's just very hard to see:

image

Looks good to me.

**Pull request merged**